recipes/inotify-tools/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "inotify-tools" %}
+{% set version = "3.14" %}
+{% set sha256 = "97a4fdf9132da47c2baa677c1a427c435ee55a26df42b2b74a41489b9fbd2323" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rvoicilas/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - automake
+    - libtool
+
+test:
+  commands:
+    - which inotifywatch
+    - which inotifywait
+
+about:
+  home: https://github.com/rvoicilas/inotify-tools
+  license: GPL-2
+  license_family: GPL
+  license_file: COPYING
+  summary: 'This is a package of some commandline utilities relating to inotify.'
+  description: |
+    The general purpose of this package is to allow inotify's features to be
+    used from within shell scripts.  Read the man pages for further details.
```

Comment:
I know you just copied this from the repo, but I don't know that "Read the man pages for more details" is really adding much to our description here. :)